CONTRIBUTING.md

```diff
@@ -1,22 +1,60 @@
-# Contributing
-So you're thinking about contributing to our sales tool, great!
-To make sure every contributor is on the same 'page', the CommIT team advises to keep the following guidelines in mind when contributing.
-
-## Branching Strategy
-For branching, we apply the same branching strategy as we use in Koala:
-- `feature/` for new features.
-- `bug/` for bugfixes.
-- `doc/` for documentation.
-- `test/` for testing.
-- `debt/` for refactoring and enhancements.
-
-## Commit Messages
-Please make sure to write descriptive commit messages:
-1. The header should be 72 characters or less.
-2. Write messages in the imperative: "Fix bug X", "Add feature Y", etc.
-3. Reference issue in commit that fixes it: "Fixes #12: Remove bug Z".
-4. Add a body to the message in which you elaborate on the header.
-
-## Thanks!
-Thank you a lot for any PR's!
+# Contributing to Magnificent Sloth
 
+Thank you for your interest in contributing to Magnificent Sloth! We welcome contributions from students, CommIT and IT Crowd members to help us improve and grow our projects.
+
+## Getting Started
+
+Before you start contributing, please make sure you have:
+
+1. Created a GitHub account: If you don't have one already, you can [sign up here](https://github.com/).
+
+2. Familiarized yourself with Git and GitHub: You may find resources like [GitHub's Guide to Git](https://docs.github.com/en/get-started/quickstart) helpful.
+
+3. Installed Git on your local machine: You can download it from [here](https://git-scm.com/).
+
+## Contributing
+
+To contribute to our projects, please follow these steps:
+
+1. **Clone the Repository**: Use `git clone` to clone the repository to your local machine.
+
+```shell
+git clone https://github.com/svsticky/radio.git
```

Comment:
Update this to `magnificent-sloth.git`

Reply:
Damn it, scanned too quickly. If only phones had ctrl+f